src/lib/services/strategy-schema.ts

```diff
@@ -5,6 +5,7 @@ const strategySchema = joi
     .object()
     .keys({
         name: nameType,
+        title: joi.string().default('').optional(),
```

Comment:
so do we want to persist empty string for new strategies? 

Also is we have optional and default isn't default enough?

Reply:
Actually, I will make it the same as description here, thanks for catching